src/store/store.ts

```diff
@@ -1,37 +1,119 @@
 import { AnyAction, createStore, Dispatch } from "@reduxjs/toolkit";
-import { UPDATE_STATUS } from "common/actions";
-import { UpdateStatus } from "common/models";
+import { UPDATE_STATUS, CREATE_ISSUE } from "common/actions";
+import { CreateIssue, UpdateStatus } from "common/models";
 
 const initialState = {
   issues: [
-    {id: 0, name: 'Learn Redux', description: 'Read the official docs of Redux', status: 'New', assignee: 'Assignee Name'},
-    {id: 1, name: 'Setup project', description: 'An empty React project with TS and Redux', status: 'New', assignee: 'Assignee Name'},
-    {id: 2, name: 'Implement Trello Board', description: 'A Kanban board with drag-and-drop feature', status: 'New', assignee: 'Assignee Name'},
-    {id: 3 , name: 'Submit code for review', description: 'Open a new pull request', status: 'New', assignee: 'Assignee Name'},
+    {
+      id: 0,
+      name: "Learn Redux",
+      description: "Read the official docs of Redux",
+      status: "New",
```

Comment:
The status enum can be used in this file

Reply:
done